### PR TITLE
Code bugs.

### PR DESCRIPTION
**Pull request by RusTit**
At first i thought that something is broken, but after some tests I realized that everything work (flower image sizes). So calculation can be removed.
Fuchsia has one section.
https://github.com/ForestryMC/Binnie/blob/4946c9d31cc2c5e02eef2cd56f659ae89a402cc4/botany/src/main/java/binnie/botany/genetics/EnumFlowerType.java#L50
Foxglove - two.
https://github.com/ForestryMC/Binnie/blob/4946c9d31cc2c5e02eef2cd56f659ae89a402cc4/botany/src/main/java/binnie/botany/genetics/EnumFlowerType.java#L37

### CODE REVIEW

**Comment by RusTit** [comment]
From Idea:
Reports a null check followed by an instanceof check or a method call which will surely return false when null is passed (e.g. Class.isInstance). Since the instanceof operator always returns false for null, there is no need to have an additional null check.
Here is an example of a violation:
`    if (x != null && x instanceof String) { ... }`
The quickfix changes this code to:
`    if (x instanceof String) { ... }`

**Comment by RusTit** [comment]
@Nedelosk I will be grateful if you review this and #504 today

File: core/src/main/java/binnie/core/util/EmptyHelper.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package binnie.core.util;
+
+public final class EmptyHelper {
+    private EmptyHelper() {}
+
+    public static final String EMPTY_STRING = "";
```
Comment (mezz) [inline core/src/main/java/binnie/core/util/EmptyHelper.java:6]: I think this is probably overkill, I'd be surprised if the JVM does not have some kind of optimization for the empty string.

Comment (RusTit) [reply core/src/main/java/binnie/core/util/EmptyHelper.java:6]: Hm. Probably you are right.
But now I used StringUtils.Empty (as you can see in #504)

Comment (mezz) [reply core/src/main/java/binnie/core/util/EmptyHelper.java:6]: Ok, sounds good.

**Pull request merged**